app/viewmodels/authorisedOfficials/AddedOneAuthorisedOfficialHelper.scala

```diff
@@ -89,9 +89,14 @@ class AddedOneAuthorisedOfficialHelper(index: Index, mode: Mode, countryService:
                   messagePrefix = "authorisedOfficialAddress")
 
   def authOfficialHadPreviousAddressRow: Option[SummaryListRow] =
-    answerPrefix(AuthorisedOfficialPreviousAddressPage(index),
+    answerPrefix(IsAuthorisedOfficialPreviousAddressPage(index),
                  authOfficialRoutes.AuthorisedOfficialPreviousAddressController.onPageLoad(mode, index),
-                 messagePrefix = "authorisedOfficialPreviousAddress")
+                 messagePrefix = "isAddAnotherAuthorisedOfficial")
```

Comment:
need to change messages - that labels the summary as 'Add another authorised official'

Reply:
it is naming convetion conflicts between authorisedOfficialPreviousAddress and isAuthorisedOfficialPreviousAddress, doen the rename for cotrollers as well